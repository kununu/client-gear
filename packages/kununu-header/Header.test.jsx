import React from 'react';
import renderer from 'react-test-renderer';

import {Header, HeaderNav, HeaderNavItem} from '.';

test('Renders Logo without crashing', () => {
  const component = renderer.create(
    <Header title="Volle Transparenz am Arbeitsmarkt">
      <HeaderNav>
        <HeaderNavItem>
          <a href="">
            <span className="hidden-xs"><i className="fa fa-search hidden-xs" aria-hidden="true" />&nbsp; Suchen</span>
            <i className="fa fa-search visible-xs" aria-hidden="true" />
          </a>
        </HeaderNavItem>
      </HeaderNav>
    </Header>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

