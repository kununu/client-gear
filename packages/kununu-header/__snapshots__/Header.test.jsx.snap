exports[`test Renders Logo without crashing 1`] = `
<header
  className="header fixed"
  role="banner">
  <div
    className="container-fluid">
    <div
      className="flex">
      <div
        className="pullLeft">
        <div
          className="logoContainer">
          <a
            href="">
            <div
              className="logo light">
              <h1
                className="sr-only">
                kununu
              </h1>
              <span
                aria-hidden="true"
                className="starSpinner light"
                style={
                  Object {
                    "transform": "rotate(0deg)",
                  }
                } />
            </div>
          </a>
        </div>
        <span
          className="title">
          Volle Transparenz am Arbeitsmarkt
        </span>
      </div>
      <div
        className="pullRight">
        <nav
          className="headerNav">
          <ul>
            <li
              className="headerNavItem">
              <a
                href="">
                <span
                  className="hidden-xs">
                  <i
                    aria-hidden="true"
                    className="fa fa-search hidden-xs" />
                  Â  Suchen
                </span>
                <i
                  aria-hidden="true"
                  className="fa fa-search visible-xs" />
              </a>
            </li>
            <li
              className="headerNavItem">
              <a
                className=""
                href="/">
                <span
                  className="hidden-xs">
                  Mein kununu 
                </span>
                <i
                  className="fa fa-user visible-xs" />
              </a>
            </li>
            <li
              className="headerNavItem">
              <a
                className="btn btn-dd-sm btn-primary"
                href="/">
                Firma bewerten
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</header>
`;
